analyzer/consensus/consensus.go

```diff
@@ -80,6 +82,9 @@ func NewAnalyzer(cfg *config.BlockBasedAnalyzerConfig, genesisChainContext strin
 
 // Implements BlockProcessor interface.
 func (m *processor) PreWork(ctx context.Context) error {
+	// Start background goroutine to fetch current node height.
+	go m.fetchNodeHeights(ctx)
```

Comment:
Does it make more sense to start this later, e.g. at the end of `PreWork` after genesis is processed?

Reply:
moved to separate analyzer!